REST/PowerShell/Feeds/SyncPackages.ps1

```diff
@@ -119,9 +119,11 @@ $destinationSpaceId = ((Invoke-RestMethod -Method Get -Uri "$destinationOctopusU
 # Create HTTP clients 
 $sourceHttpClient = New-Object System.Net.Http.HttpClient
 $sourceHttpClient.DefaultRequestHeaders.Add("X-Octopus-ApiKey", $sourceOctopusAPIKey)
+$sourceHttpClient.Timeout = New-TimeSpan -Minutes 60
```

Comment:
Can we put the `60` into a variable used by both timeouts so it can easily be changed by the user

Reply:
Personally I wouldn't bother with making a CLI param.  
I probably wouldn't bother lowering it, and if my packages are bigger than that, then... we might have other problems :|